download_test_vectors.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+dl_version() {
+	[[ -z "$1" ]] && { echo "usage: dl_version() vX.Y.Z"; exit 1; }
+
+  [[ -d "tarballs/$1" ]] || {
+    mkdir -p "tarballs/$1"
+    pushd "tarballs/$1"
+    curl -L --remote-name-all "https://github.com/ethereum/eth2.0-spec-tests/releases/download/$1/{general,minimal,mainnet}.tar.gz"
+    popd
+  }
+}
+
+unpack_version() {
+	[[ -z "$1" ]] && { echo "usage: unpack_version() vX.Y.Z"; exit 1; }
+
+  [[ -d "tests-$1" ]] || {
+    cat "tarballs/$1"/{general,minimal,mainnet}.tar.gz | tar --one-top-level="tests-$1" --strip-components 1 -xvzf - -i
+  }
+}
+
+dl_version v0.8.3
```

Comment:
Version 0.8.3, not 0.8.4? It shouldn't make a difference, since they're nigh-identical.

Reply:
because that was what the eth2.0-spec-tests submodule had.